Zilon.Core/Zilon.Core.Benchmarks.Move/Program.cs

```diff
@@ -1,16 +1,12 @@
 ﻿using BenchmarkDotNet.Running;
 
-using Zilon.Core.Benchmark;
-using Zilon.Core.Benchmarks.Common;
-
-namespace Zilon.Core.Benchmarks
+namespace Zilon.Core.Benchmarks.Move
 {
-    static class Program
+    class Program
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310892)